bindings/wasm/src/iota/iota_document.rs

```diff
@@ -591,20 +584,20 @@ impl From<WasmIotaDocument> for IotaDocument {
 
 #[wasm_bindgen]
 extern "C" {
-  #[wasm_bindgen(typescript_type = "IotaDIDUrl | string")]
-  pub type UIotaDIDUrlQuery;
+  #[wasm_bindgen(typescript_type = "DIDUrl | string")]
+  pub type UDIDUrlQuery;
 
   #[wasm_bindgen(typescript_type = "IotaDID[]")]
   pub type ArrayIotaDID;
 
   #[wasm_bindgen(typescript_type = "IotaDocument[]")]
   pub type ArrayIotaDocument;
 
-  #[wasm_bindgen(typescript_type = "IotaService[]")]
-  pub type ArrayIotaService;
+  #[wasm_bindgen(typescript_type = "Service[]")]
+  pub type ArrayService;
 
-  #[wasm_bindgen(typescript_type = "IotaVerificationMethod[]")]
-  pub type ArrayIotaVerificationMethods;
+  #[wasm_bindgen(typescript_type = "VerificationMethod[]")]
+  pub type ArrayVerificationMethods;
```

Comment:
Some of these types (like these two right here) are now duplicated across `iota_document.rs` and `wasm_core_document.rs`, perhaps we can move them into `common/types.rs`?

Reply:
Nice catch!